COMETwebapp/IterationService/IterationService.cs

```diff
@@ -0,0 +1,111 @@
+﻿namespace COMETwebapp.IterationService
```

Comment:
- missing code header
  - there does not seem to be a unit test for this class, please add